policyengine_us/parameters/gov/states/ky/tax/income/credits/family_size/percentage.yaml

```diff
@@ -0,0 +1,57 @@
+description: Kentucky allows for the family size tax credit percentage, based on modified gross income as a percent of the federal poverty guidelines.
```

Comment:
What's it a percentage of? Can we add a variable for the actual value of the credit?

Reply:
added as a refundable credit